client/package.json

```diff
@@ -7,28 +7,56 @@
     "build:dev": "webpack --mode development",
     "run:prod": "webpack-dev-server --mode production",
     "run:dev": "webpack-dev-server --mode development",
-    "test": "jest --verbose",
-    "test:coverage": "jest --verbose --coverage"
+    "test": "jest",
+    "test:coverage": "jest --coverage"
   },
   "dependencies": {
-    "jest": "^24.5.0",
+    "keymirror": "^0.1.1",
+    "prop-types": "^15.7.2",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-redux": "^6.0.1",
     "react-router": "^4.4.0",
     "redux": "^4.0.1"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.4",
-    "babel-preset-react": "^6.24.1",
+    "@babel/core": "^7.4.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.4.3",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.5",
     "css-loader": "^2.1.1",
+    "cypress": "^3.2.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.12.0",
     "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "file-loader": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
+    "jest": "^24.7.1",
+    "react-test-renderer": "^16.8.6",
     "style-loader": "^0.23.1",
     "webpack": "^4.29.6",
     "webpack-cli": "^3.3.0",
     "webpack-dev-server": "^3.2.1"
+  },
+  "jest": {
```

Comment:
_trivial_

_from the task docs:_
> Coverage > 60%

I think, that to be sure that your code always meets this requirement, you can introduce the `threshold` property of the `jest` config like so

```
  "jest": {
    "coverageThreshold": {
      "global": {
        "branches": 60,
        "functions": 60,
        "lines": 60,
        "statements": -10
      }
    }
  }
```

PS Just keep in mind this feature if might need it a real project to keep up maintaining decent code coverage.
PSS In the test project it is up to you to apply this option or not.